rvepak/mailroom.py

```diff
@@ -0,0 +1,137 @@
+import os
+import sys
+import prettytable
+
+donors = [('Jack', 1), ('John', 2), ('Ray', 3), ('Kelly', 2), ('Rick', 1)]
+
+maxTriesCount = 0
+
+
+def inputPrompts():
+    '''
+    this method prompts for the 3 values
+    '''
+    global maxTriesCount
+    allowedResponsesList = allowedResponses(allowedResponsesMap)
+    response = input('Choose an action from the following {}: '.format(
+        allowedResponsesList))
+    if str(response) not in allowedResponsesList:
+        while maxTriesCount < 3:
```

Comment:
I love that there are a limited number of attempts before getting booted. Good security feature!